EKF/ekf.h

```diff
@@ -300,6 +311,9 @@ class Ekf : public EstimatorInterface
 	// fuse airspeed measurement
 	void fuseAirspeed();
 
+	// fuse sideslip measurement
```

Comment:
Make it clear in the comment that this is not a real measurement, eg 'fuse synthetic zero sideslip measurement'